packages/components/pie-button/custom-elements-manifest.config.js

```diff
@@ -1,3 +1,3 @@
-import customElementsManifestConfig from '@justeattakeaway/pie-components-config/custom-elements-manifest.config.js';
+import customElementsManifestConfig from '@justeattakeaway/pie-components-config/custom-elements-manifest.config';
```

Comment:
Is this file still exported or do we now need to do a named import from the main entry point?